### ISSUE TITLE
Why this error pops out ? trying to use the built in webcam from my laptop

### ISSUE THREAD

**Issue by klniceajer** [nhtua/greencam#13]
Couldn't get the webcam object! Could not start video source

**Comment by nhtua** [comment]
What version you were using? Is it greencam.netlify.app? It would be useful if you gave a screenshot.

**Comment by ItachiSan** [comment]
Hi @nhtua , I have the same issue so I can report it here. :)
I use OBS Studio latest version (pre 27.0) on Windows 10.
I get the same issue when using the app as a Browser Source.

I believe the documentation is incorrect and that the source is another. :)

**Comment by klniceajer** [comment]
Yeah i was using the site and this error pops out  Couldn't get the webcam
object! Could not start video source
Okay doesnt work on chrome / edge but does on firefox
[image: image.png]
